src/filetreediff.js

```diff
@@ -197,10 +198,28 @@ export class FileTreeDiffElement extends LitElement {
           ${this.renderDocDiff()}
           <select @change=${this.handleFileChange}>
             <option value="" ?selected=${!hasCurrentFile} disabled>
-              Files changed
+              Files changed in #${this.config.versions.current.slug}
             </option>
             ${renderSection(diffData.added, "Added")}
             ${renderSection(diffData.modified, "Changed")}
+            <optgroup label="Links">
+              <option
+                value="${addUtmParameters(
+                  this.config.builds.current.urls.build,
+                  "notification",
+                )}"
+              >
+                Go to build log
```

Comment:
The verb seems unneeded on these. These would also feel more natural under a dedicated menu, which would remove the need to explain that these are links that go to another page.

```suggestion
                Build log
```